arduino-ide-extension/package.json

```diff
@@ -48,14 +48,15 @@
     "@types/which": "^1.3.1",
     "ajv": "^6.5.3",
     "async-mutex": "^0.3.0",
+    "axios": "^0.21.1",
```

Comment:
@AlbyIanna I love axios, but are we sure we want to add another dependency when we have something somehow similar? (`"request": "^2.82.0"` should be included already)

Reply:
No, you're right. I initially decided to use axios before the API was returning 403 and using `request` I couldn't handle the response properly, but now the API is not returning 403 anymore so I can change this.